experimental/sev_guest/src/guest.rs

```diff
@@ -62,13 +73,17 @@ pub struct GuestMessageHeader {
     /// Reserved. Must be zero.
     _reserved_0: u64,
     /// The algorithm used to encrypt the payload.
-    pub algorithm: AeadAlgorithm,
+    ///
+    /// Use `get_algorithm` to try to convert this to an `AeadAlgorithm` enum.
```

Comment:
```suggestion
    /// Use [`GuestMessageHeader::get_algorithm`] to try to convert this to an [`AeadAlgorithm`] enum.
```

so that if names change it will be detected.

(also below)

Reply:
Done throughout.